db/migrate/20150705131449_add_type_to_reports.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeToReports < ActiveRecord::Migration
+  def change
+    add_column :reports, :type, :string, :default => 'PuppetReport'
```

Comment:
two issues:
1. what about salt/chef reports? they would also be migrated to puppet report?
2. type should be null => false